src/backends/latex/types.jl

```diff
@@ -41,6 +42,7 @@ This structure defines the format of the LaTeX table.
     subheader_envs::Vector{String}       = ["texttt"]
     hlines::Vector{Symbol}               = [:begin,:header,:end]
     vlines::Union{Symbol,Vector{Symbol}} = :none
+    wrap_table::Bool                      = true
```

Comment:
I do not think we need this here. IMHO, this is not related to the format of the table, just the user option to wrap or not the table inside the environment `table`.

Reply:
Sorry! I understand better now.